src/Configurator/BundlesConfigurator.php

```diff
@@ -90,6 +90,14 @@ private function dump(string $file, iterable $bundles): void
         }
 
         file_put_contents($file, $contents);
+
+        if (function_exists('opcache_invalidate')) {
+            opcache_invalidate($file);
+        }
+
+        if (function_exists('apc_delete_file')) {
+            apc_delete_file($file);
```

Comment:
this looks useless. Flex requires PHP 7.1+, and the APC extension exists only for PHP 5.5-